pkg/controller/certificates/sync.go

```diff
@@ -121,13 +142,44 @@ func (c *Controller) Sync(ctx context.Context, crt *v1alpha1.Certificate) (err e
 
 	crtCopy := crt.DeepCopy()
 
+	// get issuer details for metrics
+	issuerName := issuerObj.GetObjectMeta().Name
+	var issuerScope string
+	switch crt.Spec.IssuerRef.Kind {
+	case v1alpha1.IssuerKind:
+		issuerScope = "namespaced"
+	case v1alpha1.ClusterIssuerKind:
+		issuerScope = "cluster"
```

Comment:
I'd rather log this as `issuer_kind` and `ClusterIssuer` or `Issuer` as the value